05-onchain.md

```diff
@@ -89,15 +90,19 @@ trigger any action.
 # Commitment Transaction
 
 The local and remote nodes each hold a *commitment transaction*. Each of these
-commitment transactions has four types of outputs:
+commitment transactions has six types of outputs:
 
 1. _local node's main output_: Zero or one output, to pay to the *local node's*
-commitment pubkey.
+delayed pubkey.
 2. _remote node's main output_: Zero or one output, to pay to the *remote node's*
-commitment pubkey.
-3. _local node's offered HTLCs_: Zero or more pending payments (*HTLCs*), to pay
+delayed_pubkey.
```

Comment:
nit: it's `delayed_pubkey` here, but `delayed pubkey` (no underscore) the line above

Reply:
fixed